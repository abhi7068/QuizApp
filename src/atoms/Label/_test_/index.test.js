import React from 'react';

import Label from '../../Label/index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<Label />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<Label />);
  expect(asFragment(render(<Label />))).toMatchSnapshot();
});
