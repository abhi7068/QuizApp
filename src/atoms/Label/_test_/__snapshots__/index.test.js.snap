// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <label
    class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
  />
</DocumentFragment>
`;
